### PR TITLE
[react-native-vector-icons] Add loadFont() method to Icons

### PR DESCRIPTION
**Pull request by timwangdev**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/oblador/react-native-vector-icons/blob/master/lib/create-icon-set.js#L149-L160>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by timwangdev** [comment]
This API change is firstly introduced in [4.2](https://github.com/oblador/react-native-vector-icons/releases/tag/v4.2.0).
I also bumped the version to latest 4.3.
Please have a review @iRoachie .

File: types/react-native-vector-icons/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-// Type definitions for react-native-vector-icons 4.2
+// Type definitions for react-native-vector-icons 4.3
 // Project: https://github.com/oblador/react-native-vector-icons
 // Definitions by: Kyle Roach <https://github.com/iRoachie>
+//                 Tim Wang <https://github.com/timwangdev>
```
Comment (iRoachie) [inline types/react-native-vector-icons/index.d.ts]: Adding your name here means you sign up as a maintainer, and you'll be notified of all issues, PRs for the library. Just making sure you know

Comment (timwangdev) [inline types/react-native-vector-icons/index.d.ts]: Yes. I understand.

**Comment by iRoachie** [comment]
👍

**Pull request merged**